Added missing Norwegian translations

strandolsen opened the pull request.

strandolsen committed:
Added missing Norwegian translations

michaelherger commented on Slim/Plugin/LibraryDemo/strings.txt:
```diff
@@ -1,15 +1,19 @@
 PLUGIN_LIBRARY_DEMO_DESC
 	EN	Plugin to demonstrate support for multiple (or virtual) libraries. This plugin will create some (partly useless) virtual libraries. It's mostly intended as an inspiration for plugin developers.
 	NL	Plug-in om de ondersteuning van meerdere (of virtuele) collecties te demonstreren. Deze plug-in maakt (deels nutteloze) virtuele collecties aan. Het is meer bedoelt als een inspiratiebron voor plug-inontwikkelaars.
+	NO	Plugin for � demonstrere st�tte for multiple (eller virtuelle) samlinger. Denne modulen lager noen (delvis ubrukelige) virtuelle samlinger, og er mest tenkt som en inspirasjon for plugin-utviklere.
```
Here's another problem with the file's encoding. Is it still set to UTF8?

michaelherger commented on Slim/Plugin/ACLFiletest/strings.txt:
```diff
@@ -1,7 +1,9 @@
-	NL	Deze plug-in maakt het op sommige systemen mogelijk om bestandsrechten op basis van ACL's te controleren. Schakel deze plug-in alleen in als je zeker weet dat je systeem ACL's and Perl's 'use filetest' ondersteunt.
\ No newline at end of file
+	NL	Deze plug-in maakt het op sommige systemen mogelijk om bestandsrechten op basis van ACL's te controleren. Schakel deze plug-in alleen in als je zeker weet dat je systeem ACL's and Perl's 'use filetest' ondersteunt.
+	NO	Denne plugin-modulen lar deg kontrollere filtilganger basert p� ACL'er p� noen systemer. Ikke aktiver denne modulen med mindre du vet at systemet ditt st�tter ACL'er og Perls 'use filetest'.
```
Looks like the encoding of this file isn't correct. Make sure your text editor can handle UTF-8 (and does so).

strandolsen replied:
Ah, that's the penalty for being lazy and using Notepad.

I'll fire up Emacs and fix the encoding tomorrow (hopefully).

2016-08-30 20:23 GMT+02:00 michaelherger notifications@github.com:

> In Slim/Plugin/ACLFiletest/strings.txt
> https://github.com/Logitech/slimserver/pull/110#discussion_r76851326:
> 
> > \ No newline at end of file
> > -   NL  Deze plug-in maakt het op sommige systemen mogelijk om bestandsrechten op basis van ACL's te controleren. Schakel deze plug-in alleen in als je zeker weet dat je systeem ACL's and Perl's 'use filetest' ondersteunt.
> > -   NO  Denne plugin-modulen lar deg kontrollere filtilganger basert p� ACL'er p� noen systemer. Ikke aktiver denne modulen med mindre du vet at systemet ditt st�tter ACL'er og Perls 'use filetest'.
> 
> Looks like the encoding of this file isn't correct. Make sure your text
> editor can handle UTF-8 (and does so).
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/Logitech/slimserver/pull/110/files/3b71770242024bf1d6354065b1cec89c832b781c#r76851326,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AQYZHheanNn0zCZEjysIK_9lbA62W-PEks5qlHU0gaJpZM4JwxYj
> .

michaelherger commented on Slim/Plugin/SpotifyLogi/Plugin.pm:
```diff
@@ -86,7 +86,9 @@ sub initPlugin {
 	);
 	
 	# in LMS we're going to authenticate every player - don't do this on mysb, it's too expensive
-	Slim::Plugin::SpotifyLogi::ProtocolHandler->init();
```
This shouldn't be part of your pull request... Make sure only the text files (and others you might have changed) are part of it.

mherger replied:
I don't know what exactly happened there. But maybe you should have used "git pull --rebase" to make sure your changes are applied on top of the latest code. Hard to tell...

michaelherger commented on Slim/Plugin/SpotifyLogi/ProtocolHandler.pm:
```diff
@@ -19,7 +19,7 @@ my $log = Slim::Utils::Log->addLogCategory( {
 	'description'  => 'PLUGIN_SPOTIFYLOGI_MODULE_NAME',
 } );
 
-sub init {
```
See above: not part of your change. Please revert this file.

michaelherger commented on Slim/Plugin/WiMP/strings.txt:
```diff
@@ -1,10 +1,12 @@
+﻿PLUGIN_WIMP_MODULE_NAME
 	EN	TIDAL
 	NL	TIDAL
+	NO	TIDAL
```
No need to repeat strings which are identical with the English term. Just leave it alone - LMS would automatically use the identical English version as a fallback. Saving a few bytes :-)

strandolsen commented on Slim/Plugin/SpotifyLogi/Plugin.pm:
```diff
@@ -86,7 +86,9 @@ sub initPlugin {
 	);
 	
 	# in LMS we're going to authenticate every player - don't do this on mysb, it's too expensive
-	Slim::Plugin::SpotifyLogi::ProtocolHandler->init();
```
How do I limit the pull request to only certain files? I assumed that the
GIT system only merged changed files...

Magnar.

2016-08-30 20:25 GMT+02:00 michaelherger notifications@github.com:

> In Slim/Plugin/SpotifyLogi/Plugin.pm
> https://github.com/Logitech/slimserver/pull/110#discussion_r76851584:
> 
> > @@ -86,7 +86,9 @@ sub initPlugin {
> >     );
> > 
> > ```
> > # in LMS we're going to authenticate every player - don't do this on mysb, it's too expensive
> > ```
> > -   Slim::Plugin::SpotifyLogi::ProtocolHandler->init();
> 
> This shouldn't be part of your pull request... Make sure only the text
> files (and others you might have changed) are part of it.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/Logitech/slimserver/pull/110/files/3b71770242024bf1d6354065b1cec89c832b781c#r76851584,
> or mute the thread
> https://github.com/notifications/unsubscribe-auth/AQYZHth4tGgNWj0N8MafgppbSaQruSejks5qlHWJgaJpZM4JwxYj
> .

strandolsen committed:
Changed coding system from utf-8-dos to utf-8-unix

I used Notepad to edit the files, which probably messed up the coding
system. I therefore opened the files in Emacs and changed the coding
system to utf-8-unix. Hopefully that fixes it.

strandolsen committed:
Spotify plugin: Replaced outdated files with correct versions

strandolsen committed:
Discovered a small error in translation

mherger commented:
Thanks you _very_ much!
